.requirements/ghc/ubuntu

```diff
@@ -0,0 +1 @@
+autoconf automake curl g++ gcc git libgmp-dev libllvm5.0 libtinfo-dev libtool llvm-5.0 llvm-5.0-dev make ncurses-dev python3 realpath xz-utils
```

Comment:
I wonder about `llvm`: the versions vary with each GHC, and most users don't need it. I'd drop that.

Also it seems that these are requirements to **build** the ghc, not only use it. Though most are reasonable to have. (E.g. `git` is quite optional :)